themes/custom/az_barrio/css/style.css

```diff
@@ -399,16 +399,21 @@ padding: 0.625rem .75rem;
   height: 43px;
 }
 
-.search-block-form .input-group-append button {
+form.search-block-form .input-group .input-group-append button:hover,
+form.search-block-form .input-group .input-group-append button:focus {
+    background-color: #8B0015;
+    color: #ffffff;
+}
+
+.az-search-block .search-block-form .input-group-append button {
   font-size: 20px;
   color: #8B0015;
   background-color: #ffffff;
   border: 1px solid #ced4da;
   padding: 0 8px;
 }
 
-form.search-block-form .input-group .input-group-append button:hover,
-form.search-block-form .input-group .input-group-append button:focus {
+.az-search-block button {
```

Comment:
These properties are a duplicate of lines above, 404-405. I'm not sure why the original lines were removed, but they should be re-added and this new line added to the selector list. Lines 402-406 here should be removed.

The `az-search-block` selector could be added to the button:hover and button:focus selectors to be consistent with `.az-search-block .search-block-form .input-group-append button`.